### ISSUE TITLE
How to get rid of this history of searched items. Because some private (Hidden) apps also showing after opening this search. I want to hide the history of searched items can anyone help me.

### ISSUE THREAD

**Issue by due3die** [BranchMetrics/sesame_issues#267]
Hi,
Hope you guys are doing good!
How to get rid of this history of searched items. Because some private (Hidden) apps also showing after opening this search. I want to hide the history of searched items can anyone help me.
Thanks,
Sudharsan M

![IMG_20210912_145338](https://user-images.githubusercontent.com/3252373/132982546-c1c885fb-2901-46e1-86a4-e3ce62f68a84.jpg)

**Comment by Obegg** [comment]
Duplicate of #17 
Also - I want to remove it too.

**Comment by tannersands** [comment]
Yes, this is needed. Especially for browser apps, like DuckDuckGo, I don't want to see each search I make create a sesame shortcut for the app.
